04-gas/gas-section-five/gas-section-five.css

```diff
@@ -1,59 +1,147 @@
+   "gas-section-five-image-container";
+  row-gap: var(--subcontainer-row-gap-mobile);
+  column-gap: none;
+  padding: var(--subcontainer-padding-mobile);
+  border: var(--subcontainer-border-mobile);
+  background-color: var(--level2-background-color);
+}
 
+@media (min-width:600px) {
+  .gas-section-five-subcontainer{
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-areas:
+    "gas-section-five-copy-container gas-section-five-image-container";
+    column-gap: var(--subcontainer-column-gap-desktop);
+    padding: var(--subcontainer-padding-desktop);
+  }
 }
 
 .gas-section-five-copy-container {
-    grid-area: gas-section-five-copy-container;
-    display: grid;
-    justify-content: center;
+  grid-area: gas-section-five-copy-container;
+  display: grid;
+  align-content: start;
+  row-gap: var(--copy-container-row-gap-mobile);
+  background-color: var(--level3-background-color);
+}
+
+@media (min-width: 600px) {
+  .gas-section-five-copy-container {
     align-content: center;
-    text-align: center;
-    row-gap: var(--items-gap-mobile);
+    row-gap: var(--copy-container-row-gap-desktop);
+  }
 }
+
+.gas-section-five-title-container{
   
+}
+
+@media (min-width: 600px){
+  .gas-section-five-title-container{
+  }
+}
+
 .gas-section-five-title {
-    font-size: var(--title-size-mobile);
-    background-color: var(--title-background-color-mobile);
+  display: flex;
+  flex-direction: column;
+  row-gap: var(--title-container-row-gap-mobile);
+  text-align: center;
+  font-size: var(--gas-title-font-size-mobile);
+  font-weight: var(--gas-title-font-weight-mobile);
+}
+
+@media (min-width: 600px) {
+  .gas-section-five-title {
+    flex-direction: row;
+    column-gap: var(--title-container-column-gap-desktop);
+    row-gap: none;
+    text-align: left;
+    font-size: var(--gas-title-font-size-desktop);
+    font-weight: var(--gas-title-font-weight-desktop);
+  }
+}
+
+.gas-section-five-title-span-number{
+}
+
+.gas-section-five-title-span-title{
+}
+
+.gas-section-five-paragraph-container {
+  display: grid;
+  row-gap: var(--paragraph-container-row-gap-mobile);
+}
+
+@media (min-width: 600px) {
+  .gas-section-five-paragraph-container {
+  }
 }
 
 .gas-section-five-paragraph {
-    text-align: justify;
-    font-size: var(--content-size-paragraph-mobile);
+  text-align: center;
+  font-size: var(--paragraph-font-size-mobile);
+  font-weight: var(--paragraph-font-weight-mobile);
 }
-  
-.gas-section-five-img-container {
-   grid-area: gas-section-five-img-container;
-   display: grid;
-   align-content: center;
+
+@media (min-width: 600px) {
+  .gas-section-five-paragraph {
+    text-align: left;
+    font-size: var(--paragraph-font-size-desktop);
+    font-weight: var(--paragraph-font-weight-desktop);
+  }
+}
+
+.gas-section-five-image-container {
+  grid-area: gas-section-five-image-container;
+  display: grid;
+  align-content: start;
+  background-color: var(--level3-background-color);
+  justify-content: center; 
```

Comment:
put these in the order :

justify-content
align-content
background-color

Reply:
done